addons/mrp/tests/test_workorder_operation.py

```diff
@@ -827,3 +827,26 @@ def test_04b_test_planning_date(self):
         # Check that the workorder is planned now and that it lasts one hour
         self.assertEqual(workorder.date_planned_start, date_start, msg="Workorder should be planned tomorrow.")
         self.assertEqual(workorder.date_planned_finished, date_start + timedelta(hours=1), msg="Workorder should be done one hour later.")
+
+    def test_change_production_1(self):
```

Comment:
test_change_production_quantity_1
""" Change the quantity to produce on the MO while workorders are already planned. """